cmake/singularity_utils.cmake

```diff
@@ -0,0 +1,55 @@
+#------------------------------------------------------------------------------
+# placeholder
+#------------------------------------------------------------------------------
```

Comment:
Is a placeholder needed? Also this file (and all other cmake files) should have the LANL/triad copyright notice at the top

Reply:
@mauneyc-LANL replace placeholder with copyright?